### PR TITLE
PMM-7946 Add pmm server version check and nice error to HAProxy.

### PR DESCRIPTION
**Pull request by JiriCtvrtka**

### CODE REVIEW

**Review by percona-csalguero — APPROVED** [review]

**Comment by BupycHuk** [comment]
we have version comparer in `pmm` repo, let's reuse it

**Review by BupycHuk — CHANGES_REQUESTED** [review]
we have version comparer in `pmm` repo, let's reuse it

**Review by askomorokhov — APPROVED** [review]
Looks good to me! 👍

File: helpers/helpers.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import (
+	"fmt"
+
+	"github.com/percona/pmm/version"
+
+	"github.com/percona/pmm-admin/agentlocal"
+)
+
+// HAProxyMinPMMServerVersion contains minimum version for running HAProxy.
+const HAProxyMinPMMServerVersion = "2.15.0"
```
Comment (BupycHuk) [inline helpers/helpers.go]:
```suggestion
const haProxyMinPMMServerVersion = "2.15.0"
```

**Review by BupycHuk — APPROVED** [review]

**Pull request merged**